src/main/java/com/github/manolo8/darkbot/backpage/NativeBrowserImpl.java

```diff
@@ -0,0 +1,41 @@
+package com.github.manolo8.darkbot.backpage;
+
+import com.github.manolo8.darkbot.extensions.util.Version;
+import com.github.manolo8.darkbot.gui.titlebar.BackpageTask;
+import eu.darkbot.api.managers.NativeBrowserAPI;
+import org.jetbrains.annotations.Nullable;
+
+import java.io.IOException;
+
+public class NativeBrowserImpl implements NativeBrowserAPI {
+
+    private final BackpageManager backpageManager;
+
+    public NativeBrowserImpl(BackpageManager backpageManager) {
+        this.backpageManager = backpageManager;
+    }
+
+    @Override
+    public boolean isSupported() {
+        return BackpageTask.isSupported(new Version("1.3.0"));
```

Comment:
maybe consider having some static constants like:

`OPEN_LINK_VERSION = new Version(1.3.0);`

that way everywhere that you want to check for support, it will be like: `BackpageTask.isSupported(OPEN_LINK_VERSION)` which makes it way clearer what you're checking for, and allows a kinda version-history-like mapping of "name/description" to number